lua/modules/configs/tool/nvim-tree.lua

```diff
@@ -17,6 +17,11 @@ return function()
 		respect_buf_cwd = false,
 		sort_by = "name",
 		sync_root_with_cwd = true,
+		on_attach = function(bufnr)
+			local api = require("nvim-tree.api")
+			api.config.mappings.default_on_attach(bufnr)
```

Comment:
```suggestion
			require("nvim-tree.api").config.mappings.default_on_attach(bufnr)
```

I think we can just merge these two lines without needing another local variable. Other than that this PR sounds reasonable to me.